src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -53,6 +53,8 @@
     <Compile Include="..\src\System\Text\Json\BitStack.cs" Link="BitStack.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ReferenceFromRuntime Include="Newtonsoft.Json" />
+    <ReferenceFromRuntime Include="Newtonsoft.Json" />    
+    <!-- Copy external dependency into the test output directory. -->
+    <SupplementalTestData Include="$(RuntimePath)Newtonsoft.Json.dll" />
```

Comment:
Out of curiosity, is there a benefit of not having this dependency at all in the tests? I don't think that's feasible atm, but still wondering if there would be some value.

Reply:
Not much value as Newtonsoft.Json is also part of SourceBuild. It could even be used in source build scenarios.